packages/swap/src/routes/__tests__/__snapshots__/quote.test.ts.snap

```diff
@@ -4,7 +4,7 @@ exports[`server GET /quote doesn't include boost information when disabled 1`] =
 {
   "egressAmount": "999999999999975000",
   "estimatedDurationSeconds": 1830,
-  "estimatedPrice": "0.999999999999975",
+  "estimatedPrice": "1.001001001001001001",
```

Comment:
makes sense that `estimatedPrice` is higher because we dont include ingress and egress fee now